### PR TITLE
py-dotmap

### PR DESCRIPTION
**Pull request by gnericblu**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-dotmap/package.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+class PyDotmap(PythonPackage):
+    """Dot access dictionary with dynamic hierarchy creation and ordered iteration"""
+
+    homepage = "https://pypi.org/project/dotmap/"
+    pypi = "dotmap/dotmap-1.3.30.tar.gz"
+
+    version("1.3.30", sha256="5821a7933f075fb47563417c0e92e0b7c031158b4c9a6a7e56163479b658b368")
+
+    depends_on("py-setuptools", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-dotmap/package.py:17]:
```suggestion
    depends_on("py-setuptools", type="build")
```
Not needed at run-time

File: var/spack/repos/builtin/packages/py-dotmap/package.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyDotmap(PythonPackage):
+    """Dot access dictionary with dynamic hierarchy creation and ordered iteration"""
+
+    homepage = "https://pypi.org/project/dotmap/"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-dotmap/package.py:12]: If the pypi page is the homepage you don't need to define it, `pypi` will

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Comment by alecbcs** [comment]
Going to close this PR due to a lack of follow up from review feedback. @gnericblu happy to reopen if you still want to get this merged and have the time to work on it.

**Comment by adamjstewart** [comment]
Superseded by #42665 anyway

**Pull request closed without merging**